Digipost.Api.Client.DataTypes.Tests/Appointment/AppointmentAddressTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Digipost.Api.Client.Tests.CompareObjects;
+using Xunit;
+
+namespace Digipost.Api.Client.DataTypes.Tests.Appointment
+{
+    public class AppointmentAddressTests
+    {
+        [Fact]
+        public void AsDataTransferObject()
+        {
+            var source = new AppointmentAddress("Gateveien 1", "0001", "Oslo");
```

Comment:
Dette er ikke en kjempebiggie for meg, men vi (meg og noen) hadde en runde på dette biblioteket der vi fant ut det var nice med opprettelse av magiske strenger først og så gjenbruker. Jeg synes det leser litt dårligere, men da slipper vi duplisering. Jeg stemmer for å gjøre det her også siden det ser ut til at det er det som er mest praktisert :)

Reply:
Kommenterer det forresten ikke videre. Jeg tenker det gjelder for alle tester :)

Reply:
Jeg er (som du sikkert forstår av måten jeg har gjort det på) tilhenger av å bare duplisere heller enn å øke antallet (uvesentlige) linjer i en test ved å opprette masse variabler. Det er forøvrig ikke spesielt konsekvent tidligere heller [denne testen](https://github.com/digipost/digipost-api-client-dotnet/blob/dev/Digipost.Api.Client.Send.Tests/SendDataTransferObjectConverterTests.cs#L231) gjør slik som jeg gjør her, [en annen test](https://github.com/digipost/digipost-api-client-dotnet/blob/dev/Digipost.Api.Client.Send.Tests/SendDataTransferObjectConverterTests.cs#L46) plukker ut verdiene fra `source`-objektet.

Hva er rasjonalet bak at det er nice med gjenbruk?